.travis.yml

```diff
@@ -0,0 +1,40 @@
+dist: xenial
+language: python
+python:
+  - "2.7"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+install:
+  - pip install opentracing
+  - pip install six
+  - pip install django
+  - pip install django-opentracing
+  - pip install wavefront-opentracing-sdk-python
+  - pip install flake8 flake8-colors flake8-import-order
+  - pip install pep8-naming
+  - pip install pydocstyle
+  - pip install pylint
+before_script:
+  - python -m flake8
+  - python -m pylint wavefront_sdk
+  - python -m pydocstyle
+script:
+  - if [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]; then
+      test -n "$(git diff -G version= $TRAVIS_COMMIT_RANGE setup.py)";
+    fi
+deploy:
+  - provider: pypi
+    user: wavefront-cs
+    server: https://test.pypi.org/legacy/
+    password:
+      secure: "w4j8TyINfzCRlcXiZcYj7eIcEYmYIisSHxUsXqC+7pUCFfAwOSJEQJhkYepFWxcmxOIhY5IdWOGsU1JNBkFyzS3NNlU76xvZ7Ykl5zicVd3ng6i/Pj6Eswhf2B5BgCpQwWIXAEHPQL+Q6xmY9tL4YnaLTpXsOOliltt1qdc+W31DilnCEmoFpDUK/KEQckRMvFj60mk3dsep87jxQdtuF+we0piN0JdG5KawjYbwbsLo940ZkXittRw5WKfcIwYbZo1SU8fxflvLKX48qbz6vK9+ACYL/3gav2x2zdopNx8mmK17Q5zgNKuvSm686gXZgzJGrNaSNFm3oNmRMP0dBS4ebYR+jS7ABOiT0SekwBqWkVKLEMnaJC/1k/8JuKabeXcA9mmZP/ZhmRsVMtyP449RBEQKoditMLr10PZbiz5iDu0UjqrF7yOqxwEsIUI7lBXY4r1FBxIrAD/xU8SMFurmb/qTmwB8sbmHxX/L6HBY0giiN039KqhAvcwPbSuDYchzyHouXRkhMgpI2jtpOVottEzMyJtRqaK+qV5zdm1ST6E7WqaIvEkkNM7OGw6jxexTadhQ7w4xdjHYIcgDHHFZJ3C8ZgivaiGaLNxrTT+OJ3fwc+zGcT6ndvYYGzXusre56Zal/PbdElv+sqX0sS/U5UVv9cJU5qQYIv3lpsI="
```

Comment:
We need to generate a unique secret here, otherwise it won't work

Reply:
Can you just make a commit to change? Or just send me the new secret.